website/docs/configuration/modules.html.md

```diff
@@ -423,10 +423,54 @@ deleted.
 
 ### Limitations when using module expansion
 
-Modules using `count` or `for_each` cannot pass different sets of providers to different instances.
+Modules using `count` or `for_each` cannot include configured `provider` blocks within the module.
+Only empty provider blocks, or those with only an alias defined, are allowed.
```

Comment:
We've referred to these as "proxy configuration blocks". I'm not sure if there is a link-able heading to that section which defines in more detail.

Reply:
Yeah, I'll use that term here ... I mention "proxy" later, but I think mentioning it earlier is clearer. Also upgrading that part of the docs to a heading so I can link to it.